components/simplified-grid/simplified-grid.css

```diff
@@ -0,0 +1,34 @@
+.grid-50-50 {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+}
+.grid-30-70 {
+  display: grid;
+  grid-template-columns: 30% 70%;
+}
+.grid-70-30 {
+  display: grid;
+  grid-template-columns: 70% 30%;
+}
+.grid-3-col {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
```

Comment:
repeat(4,1fr) creates 4 columns so it has to be repeat(3, 1fr) for 3 columns.

Reply:
Ohh I was trying something and forgot to remove it. Thanks for the review.